### ISSUE TITLE
Show device name on unlock screen

### ISSUE THREAD

**Issue by thisconnect** [digitalbitbox/bitbox-wallet-app#1594]
Showing the device name on the unlock screen would allow a user to know which BB02 they are using before they unlock. This is useful if someone has multiple BB02s so if they insert the wrong one they know before they try unlocking with the wrong pw.

github: https://github.com/digitalbitbox/bitbox02-firmware/issues/825

**Comment by x1ddos** [comment]
what's the difference between this and https://github.com/digitalbitbox/bitbox02-firmware/issues/825?

**Comment by x1ddos** [comment]
oh, is this about showing the name in the app?

**Comment by thisconnect** [comment]
@x1ddos yes exactly to show the name also in the app in the unlock view.
